index.html

```diff
@@ -0,0 +1,743 @@
+        <li><a href="#">Spaces</a></li>
+        <li><a href="#">Terms & Conditions</a></li>
+      </ul>
+    </nav>
+
+    <nav id="headerNavWhite">
+      <ul>
+        <li><a href="#">LATEST</a></li>
+        <li><a href="#">HARD FORK</a></li>
+        <li><a href="#">PLUGGED</a></li>
+        <li><a href="#">FUNDAMENTALS</a></li>
+        <li><a href="#">WORK 2030</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <div class="headerDivYellow">
+    It's almost Pride Week in Amsterdam <img alt="🏳️‍🌈" src="https://s.w.org/images/core/emoji/11/svg/1f3f3-fe0f-200d-1f308.svg">
+    Don't miss our events on diversity &amp; inclusion → </div>
+
+  <main>
+
+    <section id="firstSection">
+
+      <article class="firstSectionLeft">
+        <div class="firstSectionLeftLogo"></div>
+        <div class="cover-gradient"></div>
+        <div id="coverDate">Monday — July 29, 2019</div>
+        <div class="firstGridLeft">
+          <h5>INSIGHTS</h5>
+          <h2 class="firstSectionTitle">
+            <a href="https://thenextweb.com/tech/2019/07/29/slack-is-down-so-maybe-go-to-the-pub-instead/">Slack is down so maybe go to the pub instead?</a>
+          </h2>
+          <ul>
+            <li>
+              <a id="firstSectionTitleA" href="https://thenextweb.com/author/matthewhughes/">by Matthew Hughes</a>
+            </li>
+            <li>
+              <i class="icon icon-clock"></i>
+              <time datetime="2019-07-29T15:19:14+00:00" class="timeago" data-full-date="Jul 29, 2019 - 12:19">1 hour ago</time>
+            </li>
+            <li><i class="icon icon-share"></i>16</li>
+          </ul>
+        </div>
+      </article>
+
+      <article class="firstSectionRight1">
+        <div class="cover-gradient"></div>
+        <div class="firstGridRight1">
+          <h5>TECH</h5>
+          <h2 class="firstSectionTitle">
+            <a href="https://thenextweb.com/syndication/2019/07/29/hype-is-killing-ai-heres-how-can-we-can-stop-it/">Hype is killing AI – here’s how we can stop it</a>
+          </h2>
+          <ul>
+            <li><a id="firstSectionTitleA" href="https://thenextweb.com/author/ben-dickson/">by Ben Dickson</a></li>
+            <li><i class="icon icon-clock"></i><time datetime="2019-07-29T19:39:40+00:00" class="timeago" data-full-date="Jul 29, 2019 - 14:39">29 minutes ago</time></li>
+            <li><i class="icon icon-share"></i>35</li>
+          </ul>
+        </div>
+      </article>
+
+      <article class="firstSectionRight2">
+        <div class="cover-gradient"></div>
+        <div class="firstGridRight2">
+          <h5>GEAR</h5>
+          <h2 class="firstSectionTitle">
+            <a href="https://thenextweb.com/syndication/2019/07/29/hype-is-killing-ai-heres-how-can-we-can-stop-it/">Google ad confirms the Pixel 4's face unlock and handwave-y gestures</a>
+          </h2>
+          <ul>
+            <li><a id="firstSectionTitleA" href="https://thenextweb.com/author/napierLopez/">by Napier Lopez</a></li>
```

Comment:
`SUGGESTION`

You seem to be using this ID more than once, as it appears [here]()